tests/tests/cpld_bmark_p8

```diff
@@ -101,33 +99,34 @@ export OCNRES=025
 export ICERES=0.25
 export NX_GLB=1440
 export NY_GLB=1080
-export NPROC_ICE=$ICE_tasks
-export np2=`expr $NPROC_ICE / 2`
-export BLCKX=`expr $NX_GLB / $np2`
-export BLCKY=`expr $NY_GLB / 2`
-export WAVDOMAIN=gwes_30m
-export MESH_WAV=mesh.${WAVDOMAIN}.nc
-export MODDEF_WAV=mod_def.${WAVDOMAIN}
+export CICE_NPROC=$ICE_tasks
+export np2=`expr $CICE_NPROC / 2`
+export CICE_BLCKX=`expr $NX_GLB / $np2`
+export CICE_BLCKY=`expr $NY_GLB / 2`
+export WW3_DOMAIN=gwes_30m
+export MESH_WAV=mesh.${WW3_DOMAIN}.nc
```

Comment:
Do we want this to be `WW3_MESH` (from `MESH_WAV`)?

Reply:
update:  I see other components with the generic component name MESH_ICE, MESH_OCN.. so leave MESH_WAV as is?

Reply:
this does seem to be the case getting down through more files.  I can't seem to resolve this, but consider it so.

Reply:
I debated this change, but for now I've left the mesh names variables as MESH_WAV, MESH_ICE and MESH_OCN, primarily because they are used as generic component names vs model specific names. So for example, MESH_OCN can also refer to HYCOM (and not MOM).

Reply:
Okay, got it.  Thanks @DeniseWorthen